rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -115,7 +115,7 @@ def required_components(cls) -> List[Type[Component]]:
         # incorrect intent labels, used only if 'loss_type' is set to 'margin'.
         USE_MAX_NEG_SIM: True,
         # Scale loss inverse proportionally to confidence of correct prediction
-        SCALE_LOSS: True,
+        SCALE_LOSS: False,
```

Comment:
I think we should keep it `True` here

Reply:
As we discussed, results on all datasets suggest otherwise. I think it's better to keep what is better on a larger number of datasets.

Reply:
I agree for DIET, disagree here, a lot of people rely on it to to perform the same

Reply:
`1.8` had this effectively set to `False` and many users complained that it did perform well. @akelad what do you think?

Reply:
Reverted it back to `True`.